aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/actions/SignInChallengeCognitoActions.kt

```diff
@@ -64,9 +74,9 @@ object SignInChallengeCognitoActions : SignInChallengeActions {
     }
 
     fun AuthChallenge.getChallengeResponseKey() = when (ChallengeNameType.fromValue(challengeName)) {
```

Comment:
Put in extensions class

Reply:
I have added the function inside AuthChallenge class

Reply:
Create a separate class in cognito package instead moving it to AuthChallenge. AuthChallenge is in codegen and should not depend on kotlin SDK.